applications/ink-playground-maintance.md

```diff
@@ -0,0 +1,112 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Arjun Chamyal
+- Jatin Sharma
+- Radhe Shyam
+
+### Contact
+
+- **Contact Name:** Ghan Vashishtha
+- **Contact Email:** ghan@zeeve.io
+- **Website:** www.zeeve.io
+
+### Legal Structure
+
+- **Registered Address:** 1603 Capitol Ave Ste 310, Cheyenne 82001, WY
+- **Registered Legal Entity:** Zeeve Inc.
+
+### Team's experience
+
+Founded by a team of experienced professionals and entrepreneurs from industry, Zeeve's co-founders collectively have over 45+ years of experience in technology, product development, and various business verticals. Zeeve has built an enterprise-grade no-code Blockchain Infrastructure Automation platform that enables Enterprises, Blockchain Startups, Blockchain Consulting Companies and Web3 Developers to deploy Blockchain nodes and Decentralised Apps within minutes, and manage them with advanced analytics and real-time alerts. In June 2022, the Startup raised $2.65 Million in a Seed Round from Leo Capital and Blu Ventures. It plans to deploy the funds towards product development, augmenting the technology team and enhancing its reach among DApp developers and global corporations, please consider visiting our prior [work](https://www.zeeve.io/platform).
+
+### Team Code Repos
+
+- https://github.com/Zeeve-App
+- https://github.com/Zeeve-App/larch
+- https://github.com/Zeeve-App/supernets2-data-availability
+- https://github.com/Zeeve-App/supernets2-contracts
+- https://github.com/Zeeve-App/ink-playground
+- https://github.com/Zeeve-App/sawtooth-artifacts
+- https://github.com/Zeeve-App/Zeeve-read-the-docs
+
+
+GitHub accounts of all team members.
+
+- https://github.com/arjunzeeve
+- https://github.com/gnv-zeeve
+- https://github.com/radhe-zeeve
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/ghan-vashishtha 
+- https://in.linkedin.com/in/arjun-chamyal-a4466086
+- https://www.linkedin.com/in/jatin-sharma-567113211/
+
+## Maintenance Responsibilities :nut_and_bolt:
+
+We do want to host the ink playground website and along with code maintenance. To maintain the code base by reviewing and accepting our and the others contributions, we would like to have management level access to Ink Playground repository and DNS records of Ink playground.
+
+Our responsibilities:
+
+- We will maintain the project's documentation.
+- We will answer issues and discussions.
+- We will fix new bugs as we receive and help contributors who have pull requests get any needed information.
+
+Hosting:
+
+- The machine for Ink Playground using docker deployment
+  - 300 GB block storage
+  - 8 Core CPU (2Ghz)
+  - 32 GB
+  - expected 100 GB egress
+- The expected additional cost will be $240/month for Hosting
+
+### Overview
+
+- **Start Date:** 25th March 2024
+- **Sprint/Period Duration:** 4 Weeks
+- **Total Duration:** 1 year
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Max budget per sprint/period:** 6,000 USD (excluding server changers, if any)
```

Comment:
```suggestion
- **Max budget per sprint/period:** 6,000 USD (excluding server charges, if any)
```

Reply:
@radhe-zeeve Please accept the spelling mistake change. Thanks @semuelle We must have gotten confused, now I realise the amount for level is to be in total.

Do you advise that we make it 6 months or rather change the level?

Reply:
Fair question, @radhe-zeeve. With the handover of ink!, it's currently a bit of guesswork what will be needed. We could either sign a grant for a shorter period, or we wait until we get some feedback from the (new) ink! team.